test/users.routes.test.js

```diff
@@ -566,35 +566,51 @@ describe('users.routes.test.js', () => {
     });
 
     it('fails if the user already has a password', done => {
```

Comment:
So if the user already has a password we don't show the screen to edit its avatar/info after a donation?